main.py

```diff
@@ -64,57 +141,22 @@ def parse_html(dep):
     ]
     table_data = [row for row in table_data[2:] if len(row) == 7]
 
-    # Test code for writing data obtained to, uncomment when want to test
     with open("data/table_test", "w") as f:
         f.write(str(table_data))
 
-    # Get prof department
-    """
-    Note:
-
-    If a prof teaches subjects from other departments,
```

Comment:
I think this note is quite helpful, can you add it back? It isn't irrelevant anymore, is it?

Reply:
@Faizan2005